.flake8

```diff
@@ -0,0 +1,5 @@
+[flake8]
+ignore = E203, E266, E501, W503, F403, F401, W605, E402, C901
```

Comment:
W605 - ok, I didn't realize that

E501 and W503 - These came from the article I cited. So I think they probably have to do with making black and flake8 compatible with each other.

Reply:
Black breaks lines before binary operators so there will be conflicts between black and flake8 if we don't ignore W503

Reply:
E501 - removed this from the flake8 config and then I got a bunch of complaints about lines that cannot be changed

Example: `cdk/app.py`  in "stack_props"
`"project_git_url": "https://github.com/harvard-dce/zoom-recording-ingester.git",`

So I think it's better to have flake8 ignore line length and let black handle it?

Reply:
OK I can live with that.

Reply:
Playing around with E501... it's just a little annoying because flake8 will complain about comments that are too long, or something like this

```
"project_git_url": "https://github.com/harvard-dce/zoom-recording-ingester.git",
```

Where the solution to work with black and flake8 is this:
```
"project_git_url": (
        "https://github.com/harvard-dce/zoom-recording-ingester.git"
    ),
```

So not sure what we want to do here.

Reply:
or this...

```   comparison_operator=cloudwatch.ComparisonOperator.GREATER_THAN_THRESHOLD,
```
I would rather this just be on one line. Flake8 would ask you to break this up somehow.

Reply:
OK, i still want to see the complaints in my editor but i guess i can override in my vscode settings.